build.rs

```diff
@@ -0,0 +1,10 @@
+extern crate gcc;
+
+fn main() {
+    let sources = &[
+        "resources/mesos-c/utils.hpp",
+        "resources/mesos-c/scheduler_driver.cpp",
+    ];
+
+    gcc::compile_library("libmesosc.a", sources);
```

Comment:
Do we need to tell gcc to "enable" c++ compiling. I think that turn on a bunch of flags like libstdc++ linking.